### DISCUSSION TITLE
Galaxy Brain badge issue

### DISCUSSION THREAD

**Discussion by oscarxdev** [community/community#48448] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello!!
I'm trying to get the Galaxy Brain badge, i already did everything necessary to get it several times but it doesn't reach my profile.
can you tell me what i can do?

**Comment by jge162** [comment]
It is possible the badge is bugged, this happens when you answer your own discussion questions.

**Reply by oscarxdev** [reply]
Is there anything I can do to fix it?

**Reply by jge162** [reply]
I think it fixes itself eventually, maybe 24 hours or less.

**Reply by oscarxdev** [reply]
Thanks for your answer.

**Comment by melvincwng** [comment]
Hi @NetJox4, the Galaxy Brain badge cannot be unlocked if you had previously marked your own discussions/questions as answers. To be able to unlock the badge, here's what you need to do:

1. Unmark all your answers to your own discussions/questions.
2. Next, have at least 2 answers marked by other GitHub users.
3. Kindly wait for a while, and the badge should show.

Hope this clarifies and answers your question 😃 

Take care, have a good day!

**Reply by oscarxdev** [reply]
I already checked and i don't see that it has anything marked :disappointed:

**Reply by jge162** [reply]
Just wait now and it will show up for u

**Reply by melvincwng** [reply]
@NetJox4, from what I'm seeing, your badge is already displayed on your GitHub Profile.

Cheers and have a good day 😃

**Reply by jge162** [reply]
Yes. Switzerland lol

**Reply by touchgrr** [reply]
What if I deleted the repository on which I answered myself?

**Reply by oscarxdev** [reply]
> @NetJox4, from what I'm seeing, your badge is already displayed on your GitHub Profile.
> 
> Cheers and have a good day 😃

Yeah ty man

**Comment by ziadeleraky** [comment]
+1

**Comment by SmashedFrenzy16** [comment]
Maybe try to answer more discussions. If it did not work, wait for a few hours just in case it might need to load. If the issue still persists, then you would have to either contact GitHub support or just try agan some other day when the badges are more stable.

**Comment by Arv18** [comment]
Util

**Comment by Veercodeprog** [comment]
to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

**Comment by github-staff** [comment, marked as answer]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by JasonLuuk** [reply]
Hi @github-staff, 
Excuse me, does this mean the "Galaxy Brain" is "out of print"? In the future people will never be able to get a Galaxy Brain achievement again?

**Comment by Pavel-Shcherbo** [comment]
How do I get a Galaxy Brain badge if I have answered 20 discussions?
